### PR TITLE
Lagoon v221

### PR DESCRIPTION
**Pull request by tobybellwood**
This is a multi-chart release:

It is primarily for the release of Lagoon Core v2.21.0, but also includes the following updates:
* Updates Lagoon Insights remote&handler
* Updates SSH Portal and Token service
* Adds a pre-upgrade job to ensure broker feature flags are enabled
* Fixes up Keycloak URLs to take internal traffic
* Updates the versions of Kubernetes that are tested

### CODE REVIEW

File: charts/lagoon-core/templates/api.deployment.yaml [hunk 1]
```diff
@@ -153,14 +155,16 @@ spec:
             secretKeyRef:
               name: {{ include "lagoon-core.keycloak.fullname" . }}
               key: KEYCLOAK_API_CLIENT_SECRET
-        - name: KEYCLOAK_URL
+        - name: KEYCLOAK_FRONTEND_URL
           {{- if .Values.keycloakFrontEndURL }}
```
Comment (shreddedbacon) [inline charts/lagoon-core/templates/api.deployment.yaml:159]: ahh crap, these will need the `/auth` added to them like above

Comment (shreddedbacon) [reply charts/lagoon-core/templates/api.deployment.yaml:159]: https://github.com/uselagoon/lagoon-charts/commit/c84b5b3ae84911e48231aa121aac7cc7fdb0e3e5

i forgot that i was only adding to the init container, not the api :facepalm: bloody templates

File: charts/lagoon-core/ci/linter-values.yaml [hunk 2]
```diff
@@ -142,9 +142,9 @@ backupHandler:
       cpu: "10m"
 
 insightsHandler:
+  enabled: true
```
Comment (shreddedbacon) [inline charts/lagoon-core/ci/linter-values.yaml:145]: are we enabling by default now?

File: charts/lagoon-docker-host/ci/linter-values.yaml
Comment (shreddedbacon) [inline charts/lagoon-docker-host/ci/linter-values.yaml:1]: we don't use this docker-host chart in testing lagoon?

Comment (tobybellwood) [reply charts/lagoon-docker-host/ci/linter-values.yaml:1]: We don't but the kind update enabled the network policy - so this change had to modify it to allow testing

Comment (shreddedbacon) [reply charts/lagoon-docker-host/ci/linter-values.yaml:1]: easy

**Review by shreddedbacon — APPROVED** [review]
Nothing standing out to me here that is concerning. Mostly updates to testing and version bumps.

**Pull request merged**